src/main/java/org/fundacionjala/core/ui/WebDriverAction.java

```diff
@@ -92,6 +92,15 @@ public String getValueXPath(final By xpathField) {
         return driver.findElement(xpathField).getText();
     }
 
+    /**
+     * Method for do click to XPath.
+     *
+     * @param xpathField have be in format css String.
+     */
+    public void clickXPath(final By xpathField) {
+        wait.until(ExpectedConditions.presenceOfElementLocated(xpathField));
```

Comment:
The method only should wait for the "presenceOfElementLocated", for that reason, the name of the method should change:
`public void waitForPresenceOfElement`